.travis.yml

```diff
@@ -42,9 +44,9 @@ before_install:
   - sudo apt-get -y remove --purge postgis-2.2
   - sudo apt-get -y autoremove
 
-  - sudo apt-get -y install postgresql-9.5=9.5.2-3cdb2
-  - sudo apt-get -y install postgresql-server-dev-9.5=9.5.2-3cdb2
-  - sudo apt-get -y install postgresql-plpython-9.5=9.5.2-3cdb2
+  - sudo apt-get -y install postgresql-9.5=9.5.2-3cdb3
+  - sudo apt-get -y install postgresql-server-dev-9.5=9.5.2-3cdb3
```

Comment:
It would be nice to test against both pg9.5 and 10. David has some experience with it in camshaft. No strong opinion at this point, though.

Reply:
I was thinking on waiting until we have our pg10/pg9.6 and postgis 2.4 for production and set an extra test environment. No idea how to do it, but it shouldn't be difficult once the packages are ready. Same for cartodb-postgresql